### ISSUE TITLE
Let previous versions run

### ISSUE THREAD

**Issue by prodigy7** [flood-io/element#265]
Related to https://github.com/flood-io/element/issues/264:

I was forced to update my installation:
```
> INFO UPDATE AVAILABLE The latest stable version of element-cli is 1.3.1
> INFO Get it by running yarn global upgrade element-cli@latest
+0s info: workRootPath: /var/www/html/production/loadtest/services/flood.io/tmp/element-results/full/2020-09-07T123613.785Z
+0s info: testDataPath: /var/www/html/production/loadtest/services/flood.io
+0s debug: Loading test script: ./full.ts

● Script Compiler █████████████████████████ compiling (0%)  
 

> Error! An unexpected empty rejection occurred
 rejection: null
this is a bug, please report it here https://github.com/flood-io/element/issues/new?template=bug_report.md
```
Now, when I update, I have the current version, but the current version has a bug! When I rollback the version, I can't run my script as expteced because the check does always force me, to update the version. I can understand the behaviour, but in the current situation it stops everything!

Please make it possible to override the check with a element parameter! At the moment the behaviour is an extreme show stopper!

**Comment by prodigy7** [comment]
Okay, "workaround" is, installing element on that way:
```
sudo yarn global upgrade element-cli@stable
```
Then selecting last known working version. Then a upgrade message is not shown.

**Comment by phileba** [comment]
Hi @prodigy7, sorry for the late response to you. We've released the latest version 1.3.4 with some bug fixes and improved. Please upgrade to the latest version.
